catalyze.markdown

```diff
@@ -34,6 +34,12 @@ Heroku ever would, but we'll see.
    $ bin/catalyze.sh associate healthproprod onboarding01
    $ bin/catalyze.sh associate healthprostaging onboarding01
    ```
+   
+1. Fetch the new remotes so `git rev-parse prod/master` and `git rev-parse staging/master` don't fail in `deploy.sh`:
+   ```
+   $ git fetch prod/master
+   $ git fetch staging/master
```

Comment:
Should only need to do `git fetch prod` / `git fetch staging` and that will fetch all branches for the remote.